fuel/app/tests/api/v1.php

```diff
@@ -1058,6 +1058,17 @@ public function test_question_set_get()
 
 	}
 
+	public function test_question_history_get()
+	{
+		// ======= AS NO ONE ========
+		try {
+			$output = Api_V1::question_history_get(555);
+			$this->fail("Expected exception HttpNotFoundException not thrown");
+		} catch ( Exception $e) {
+			$this->assertInstanceOf('HttpNotFoundException', $e);
```

Comment:
Use `$this->expectException(\ HttpNotFoundException::class);` as it's simpler and more semantically specific